### PR TITLE
adds rate limit 10/hr or 50/day

### PR DESCRIPTION
**Pull request by beppek**

### CODE REVIEW

**Comment by beppek** [comment]
Don't merge this branch yet. I will change the default limit to much higher and set this strict limit to only apply to auth routes

**Review by ntwigs — APPROVED** [review]
:rocket: :taco: :vulcan_salute:

**Comment by beppek** [comment]
Ready to merge when frontend has finished with auth

**Pull request merged**
